fastrtpsgen/src/main/java/com/eprosima/fastrtps/fastrtpsgen.java

```diff
@@ -522,7 +522,7 @@ private Project parseIDL(String idlFilename) {
             topicann.addMember(new AnnotationMember("value", new PrimitiveTypeCode(Kind.KIND_BOOLEAN), "true"));
 
             // Create template manager
-            TemplateManager tmanager = new TemplateManager("FastCdrCommon:eprosima:Common");
+            TemplateManager tmanager = new TemplateManager("FastCdrCommon:eprosima:Common",ctx,true);
```

Comment:
This new usage of TemplateManager will receive **false** by default.

```suggestion
            TemplateManager tmanager = new TemplateManager("FastCdrCommon:eprosima:Common", ctx, false);
```